src/framework/link/Link.jsx

```diff
@@ -28,6 +34,7 @@ Link.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
   data: PropTypes.any,
+  isTargetBlank: PropTypes.bool,
```

Comment:
Actually that was my first version, but then I thought that any other than `"_blank"` wouldn't be a proper option, as they are rarely or actually never used. I'm fine with both, the call is yours :)